### ISSUE TITLE
Notify users about replies to their notes and allow them to reply

### ISSUE THREAD

**Issue by Dimitar5555** [organicmaps/organicmaps#1149]
Currently a lot of users are using Organic Maps to send notes to OSM. The problem is that some of them don't login on the OSM website therefor they don't reply to them if there are replies.
A possible workaround would be to notify users when somebody replies (and/or closes) their note and let them reply directly in the app.

**Comment by biodranik** [comment]
Good idea. Is it possible to implement in anonymous mode, without logging in?

**Comment by Dimitar5555** [comment]
It is possible to remember the id of the note but anonymous users can't reply to notes.

**Comment by biodranik** [comment]
What is the use-case then? Anonymous users can poll the note by id and get a reply from OSMer. But they can't reply back. Right?

**Comment by Dimitar5555** [comment]
You are right. Organic Maps requires users to be logged in OSM (at least this is what I've observed) to add notes which eliminates this issue.

**Comment by jwmh** [comment]
> You are right. Organic Maps requires users to be logged in OSM (at least this is what I've observed) to add notes which eliminates this issue.

@Dimitar5555 trying to help cleanup old issues— 
I’m confused about what your complaint or suggestion was, then.
It sounded as if you contradicted (or self-corrected) between your initial post, vs your last comment.
Do you remember if this is still an issue?
Can anyone confirm on latest fresh install on Android?
(i’m on iOS)

**Comment by Dimitar5555** [comment]
IIRC the problem was that some users aren't logged in their OSM account in Organic maps. When they upload a note from Organic Maps, the note is anonymous. Because of that, the note uploader won't get emails about replies.

**Comment by theadventurer62** [comment]
> IIRC the problem was that some users aren't logged in their OSM account in Organic maps. When they upload a note from Organic Maps, the note is anonymous. Because of that, the note uploader won't get emails about replies.

Perhaps it should be required to log in before notes are added. That would solve this issue.
